hw2/04-background.html

```diff
@@ -8,22 +8,39 @@
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="04-background.js"></script>
+    <style>
+        .background-card {
+          max-width: 550px;
+        }
+
+        body {
+          background-color: #6C61DB;
+        }
+
+        .form-group {
+          margin-bottom: 1.5em;
+        }
+    </style>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
-    </main>
+  <body class="background">
+    <div class="container">
+      <main class="background-card m-auto mt-4 card p-3">
+          <div class="">
+            <h1>Exercise 04 - Background</h1>
+            <p class="form-group">
+              The background color will change every few seconds (as indicated below).
+              You can stop this behavior and you can also change how often the
+              background color changes.
+            </p>
+            <div class="form-group">
+              <label>Background change interval (seconds): </label>
+              <input type="number" class="form-control" name="seconds"/>
+            </div>
+            <div class="form-group">
+              <input type="submit" onclick="handleSubmit(event)" class="submit btn btn-success" value="Start"/>
```

Comment:
I think you can remove the `onclick` event if you make sure the submit button is inside a `<form>`.

Currently, the `addEventListener` in the JS file doesn't do anything because there is no form to submit.

Reply:
Fixed this so it uses a <form> now, thanks for catching!